src/test/java/api/support/builders/LoanPolicyBuilder.java

```diff
@@ -333,10 +333,7 @@ public LoanPolicyBuilder renewWith(Period period, UUID dueDateLimitScheduleId) {
       this.renewable);
   }
 
-  public Builder renewWith(UUID fixedDueDateScheduleId) {
-    if(!Objects.equals(profile, "Fixed")) {
```

Comment:
Why has this check been removed? Are fixed due date schedules allowed for renewal when the profile is not fixed?

Reply:
reverted and created additional method for override logic